src/dotnet/Chat.UI.Blazor/Services/ChatListUI.cs

```diff
@@ -118,6 +119,7 @@ public virtual async Task<Trimmed<int>> GetUnreadChatCount(ChatListFilter filter
     }
 
     [ComputeMethod]
+    [SuppressMessage("Usage", "MA0004:Use Task.ConfigureAwait(false)")]
```

Comment:
Why do we need this SuppressMessageAttribute? Is not better to use `await ActiveChatsUI.WhenLoaded.ConfigureAwait(true)` if we need to continue after with the same synchronization context.

Reply:
done